packages/spor-input-react/src/PasswordInput.tsx

```diff
@@ -9,9 +9,19 @@ export const PasswordInput = (props: PasswordInputProps) => {
   const { t } = useTranslation();
   return (
     <InputGroup>
-      <Input {...props} type={isShowingPassword ? "text" : "password"} />
-      <InputRightElement>
-        <Button variant="ghost" type="button" px={4} onClick={onToggle}>
+      <Input
+        {...props}
+        type={isShowingPassword ? "text" : "password"}
+        pr="4.5em !important"
```

Comment:
Long story short, this is required because we need to have a higher specificity than the `FormControl input` selector. This could be fixed by adding custom styles for inputs instead of styling them through FormControls. I'll look into this at a later point.